ckanext/qdes_schema/templates/scheming/form_snippets/textarea.html

```diff
@@ -5,6 +5,7 @@
 {% else %}
     {% set default_value = data[field.field_name] if data[field.field_name]|length > 0 else field.default_value|replace('<current_year>', h.render_datetime(h.get_current_datetime(), '%Y')) %}
 {% endif %}
+{% set current_year= data[field.form_placeholder] if data[field.form_placeholder]|length > 0 else field.form_placeholder|replace('<current_year>', h.render_datetime(h.get_current_datetime(), '%Y')) %}
```

Comment:
The `current_year` variable is being set and it's not used?
Shouldn't this be setting a variable `form_placeholder` and passing it into the placeholder parameter in the textarea macro below?
```
placeholder=h.scheming_language_text(form_placeholder)
```

Reply:
missmatched the branches :/